packages/actions-global/src/actions/show-notification.ts

```diff
@@ -11,7 +11,7 @@ import { addNotification } from '@commercetools-frontend/notifications';
 import type { TShowNotification } from '../types';
 
 export default function showNotification<
-  Notification extends TShowNotification
+  Notification extends TShowNotification,
```

Comment:
It seems the latest version of prettier has the [required trailing comma](https://prettier.io/blog/2023/07/05/3.0.0.html#change-the-default-value-for-trailingcomma-to-all-11479httpsgithubcomprettierprettierpull11479-by-fiskerhttpsgithubcomfisker-13143httpsgithubcomprettierprettierpull13143-by-sosukesuzukihttpsgithubcomsosukesuzuki) by default everywhere which makes this notation even for generics.

I tried to disable the default and go back to the previous one (`es5`) but I can't make it work.

Not sure if we want to keep the new styling rule (I personally don't like it for generics) or if we should avoid it.

@commercetools/shield-team-fe what do you think?